src/info/host.rs

```diff
@@ -0,0 +1,45 @@
+use crate::errors;
+use crate::mlua;
+use crate::Inject;
+
+use mlua::prelude::*;
+
+use std::fs::read_to_string;
+
+#[derive(Clone, Debug)]
+pub(crate) struct Host {
+    pub model: String,
+    pub manufacturer: String,
+}
+
+impl Host {
+    pub fn new() -> Self {
+        Host {
+            model: read_to_string("/sys/devices/virtual/dmi/id/product_name").unwrap(),
+            manufacturer: read_to_string("/sys/devices/virtual/dmi/id/board_vendor").unwrap(),
```

Comment:
This returns the motherboard vendor -- See [here](https://github.com/K4rakara/freshfetch/commit/61aa9f8eb1a67cd5f0e620f40ff856846f9c91d3). 

On prebuilt systems like laptops, this makes sense, but on custom built desktops, this isn't always the case.

Reply:
Oh sorry I didn't know about this!
Out of topic, but is it normal that the commits about the motherboard don't appear on master?

Reply:
Should I parse the `product_name` to take the first word, and assume that it's the manufacturer name?

Reply:
I'd say probably not for parsing the host name, its probably best to just have the host name provided as a string, rather than a table.

Also, no, its not normal. I'm just bad at git, so I didn't realize that master was supposed to be the working branch. I'll clean up the git branches once we're to the next release.